android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -172,6 +179,11 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
      */
     fun canBegin(event: MotionEvent) = true
 
+    /**
+     * Checks whether handler can activate. Used by TextViewHook.
+     */
+    fun canActivate(view: View) = true
```

Comment:
Shouldn't this one check for `view.isPressed`?

Reply:
You're right, done in [df1c2aa](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/df1c2aae26e3eebd10861f3b7f57e19994458b87)